editoast/openapi.yaml

```diff
@@ -262,6 +262,50 @@ paths:
         204:
           description: No content
 
+  /infra/{id}/objects/{object_type}/:
+    get:
+      tags:
+        - infra
+      summary: ""
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: Infra id
+          required: true
+        - in: path
+          name: object_type
+          schema:
+            $ref: "#/components/schemas/ObjectType"
+          description: The type of the object
+          required: true
+      requestBody:
+        description: List of object id's
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: string
+      responses:
+        200:
+          description: No content
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    railjson:
+                      type: object
+                      description: Object properties in railjson format
+                    geometry:
+                      type: object
+                      description: Object linestring geometry in geojson format
```

Comment:
We should change that to have two fields `schematic` and another one `geographic`. Both have type geojson.